src/hooks/useSlider.js

```diff
@@ -3,16 +3,20 @@ import { useSpring } from 'react-spring'
 import { useDrag } from 'react-use-gesture'
 import { clamp } from '../helpers'
 
-export default function useSlider({ slides }) {
+export default function useSlider({ initialSlide, slides }) {
+  const slideIndex = clamp(initialSlide, 0, slides.length - 1)
+  const sliderX = window.innerWidth * slideIndex
```

Comment:
This line will break SSR as `window` is undefined on the server

Reply:
moved `window` to useSpring